build/run-e2e-tests.sh

```diff
@@ -75,12 +75,22 @@ ${KUBECTL} --context="${cluster_context}" -n kube-system scale --replicas=1 depl
 echo "###### loading coverage image"
 ${KIND} load docker-image managedcluster-import-controller-coverage --name ${CLUSTER_NAME}
 
-echo "###### deploy registration-operator"
-rm -rf "$WORK_DIR/registration-operator"
-git clone https://github.com/stolostron/registration-operator.git "$WORK_DIR/registration-operator"
-${KUBECTL} apply -k "$WORK_DIR/registration-operator/deploy/cluster-manager/config/manifests"
-${KUBECTL} apply -k "$WORK_DIR/registration-operator/deploy/cluster-manager/config/samples"
-rm -rf "$WORK_DIR/registration-operator"
+echo "###### deploy ocm"
+rm -rf "$WORK_DIR/_repo_ocm"
+
+export OCM_VERSION=main
```

Comment:
Do we need to use `latest` here?

Reply:
`main` and `latest` are the same image.

Reply:
https://github.com/open-cluster-management-io/ocm/pull/214 this PR is trying to fix the e2e issue.

Reply:
Got it.